cypress/integration/golden-path.local.ts

```diff
@@ -0,0 +1,5 @@
+import { declareGoldenPathTests } from "./golden-path.common";
+
+describe("Golden path (local)", () => {
+  declareGoldenPathTests({ workspaceSlug: "local", isLocal: true });
```

Comment:
Shouldn't `workspaceSlug =  "local"` be enough to ensure that we are in the `local` case?

Reply:
True! Done in aa17d50976ca403559dd8d457ad40bc719822d6b